### PR TITLE
Use the oshinko CLI in the s2i images

### PR DESCRIPTION
**Pull request by tmckayus**
This change uses the CLI instead of talking to oshinko-rest in
the driver pod.

### CODE REVIEW

**Comment by tmckayus** [comment]
WIP for now but up for comment

File: Dockerfile.pyspark [hunk 1]
```diff
@@ -23,6 +23,8 @@ RUN cd /opt && \
 RUN yum install -y golang python27-numpy && \
     yum clean all
 
+RUN yum install -y patch && yum clean all
```
Comment (tmckayus) [inline Dockerfile.pyspark]: this can probably come out, testing ...

**Comment by tmckayus** [comment]
This should be considered ready for prime time, no longer WIP.

**Comment by elmiko** [comment]
i tried this out locally, and aside from the issue of a missing ConfigMap when i start the pod it works. i think Trevor is going to patch -core for the maps before merging.

**Comment by tmckayus** [comment]
Patching -core won't work, chicken and egg problem, but I have another solution that I'll put up as an additional patch.  For the moment the workaround is "oc create configmap oshinko-spark-driver-config" in your project if you're not using the oshinko-deploy.sh script to set up.

**Pull request merged**